src/app/core/models/part.ts

```diff
@@ -45,4 +60,19 @@ export class Part {
   public insertAdLibPiece(adLibPiece: AdLibPiece): void {
     this.adLibPieces.push(adLibPiece)
   }
+
+  public getDuration(): number {
+    const expectedDuration = this.expectedDuration ?? 4000
```

Comment:
Any particular reason for it to default to 4000 (frames?, ms?, seconds?)?

Reply:
I have extracted the value out into the constant `DEFAULT_PART_DURATION`. Regarding the units, I think it makes sense that the default unit is milliseconds.